### DISCUSSION TITLE
Running SeaweedFS in a Single Container for GitLab Testing?

### DISCUSSION THREAD

**Discussion by Karsto-sys** [seaweedfs/seaweedfs#6105] [Q&A]
Has anyone successfully run SeaweedFS in a single container? I'm trying to use it as to save on GitLab services and would appreciate any insights on how you set it up. 

Thanks in advance!

**Comment by chrislusf** [comment]
Do you want to be clear what kind of testing? Is it for S3 API, FUSE mount, or WebDAV, etc?

**Reply by Karsto-sys** [reply]
Its a3 API, we are starting, master, filer, s3  and iam, together as service.
